src/pages/Blog/Blog.tsx

```diff
@@ -14,17 +17,7 @@ const Blog: React.FC = () => {
         {/*questionable if the search will be performed within the component pagination */}
       </div>
       <div>Pagination component</div>
-      <div>
-        <div>
-          <h2>Suscríbete</h2>
-          <p>deseas estar mas pendiente de más articulos</p>
-          <div>
-            <input type="email" />
-            <button>Suscríbete</button>
-          </div>
-        </div>
-        <div>img suscríbete</div>
-      </div>
+      <Subscribe text={text} setText={setText} />
```

Comment:
move state management  inside it sown component

Reply:
done